fdks/README.md

```diff
@@ -1,9 +1,7 @@
 ## <img src="https://fnproject.io/images/fn-300x125.png" alt="Fn Project Logo" height="21" width="50"> FDK Documentation
 
-### [fdk-go](fdk-go/README.md)
-### [fdk-java](fdk-java/README.md)
-### [fdk-node](fdk-node/README.md)
-### [fdk-python](fdk-python/README.md)
-### [fdk-ruby](fdk-ruby/README.md)
+FDKs are language kits designed to help make working with Fn easier in each language.
 
-For information on the [Fn container contract and the FDK click here.](../fn/develop/fn-format.md)
\ No newline at end of file
+Select one of the FDK folders above to view documentation on that specific FDK.
```

Comment:
the folders above have been deleted now

Reply:
well, I took them out, then added them back due to debate on where FDK docs go.